File: app/src/main/java/com/rtukpe/fixtures/ui/competition/fixtures/CompetitionFixturesMvpContract.java
package com.rtukpe.fixtures.ui.competition.fixtures;

import com.rtukpe.fixtures.ui.base.MvpPresenter;

public interface CompetitionFixturesMvpContract<V extends CompetitionFixturesMvpView> extends MvpPresenter<V> {

    void getFixtures();

}

File: app/src/main/java/com/rtukpe/fixtures/ui/competition/fixtures/CompetitionFixturesMvpContract.kt
package com.rtukpe.fixtures.ui.competition.fixtures

import com.rtukpe.fixtures.ui.base.MvpPresenter

interface CompetitionFixturesMvpContract<V : CompetitionFixturesMvpView> : MvpPresenter<V> {

    fun getFixtures(id: Int)

}
